contrib/jackc/pgx/v5/options.go

```diff
@@ -0,0 +1,33 @@
+package pgx
+
+type Option func(*tracer)
+
+func WithServiceName(name string) Option {
```

Comment:
Let's add godoc comments to all these public functions

Reply:
Updated with comments and tried to add some examples.